CMakeLists.txt

```diff
@@ -86,17 +86,17 @@ if(NOT TARGET FARGPARSE::fargparse)
 endif()
 
 
-option(USE_EXTDATA2G "Use ExtData2G" OFF)
-if(NOT TARGET YAFYAML::yafyaml)
-  if(USE_EXTDATA2G)
-     set (EXTDATA_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData Target")
-     find_package(YAFYAML REQUIRED)
-     message (STATUS "Building with ExtData2G")
-  else()
-     set (EXTDATA_TARGET "MAPL.ExtData" CACHE STRING "ExtData Target")
-     find_package(YAFYAML QUIET)
-     message (STATUS "Building with ExtData1G")
-  endif()
+option(USE_EXTDATA2G "Use ExtData2G" ON)
+if(USE_EXTDATA2G)
+  set (EXTDATA2G_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData2G Target")
+  set (EXTDATA_TARGET "MAPL.ExtData" CACHE STRING "ExtData Target")
+  find_package(YAFYAML REQUIRED)
+  message (STATUS "Building with ExtData2G")
+else()
+  set (EXTDATA2G_TARGET "" CACHE STRING "ExtData2G Target")
+  set (EXTDATA_TARGET "MAPL.ExtData" CACHE STRING "ExtData Target")
+  find_package(YAFYAML QUIET)
```

Comment:
@mathomp4  maybe this line is not even needed?
```suggestion
```

Reply:
@tclune Is yafyaml used anywhere else in MAPL outside of ExtData2G? If not, then I suppose we could clean up the logic in this CMakeLists.txt a bit more. Something like:

```cmake
set (EXTDATA_TARGET "MAPL.ExtData" CACHE STRING "ExtData Target")
option(USE_EXTDATA2G "Use ExtData2G" ON)
if(USE_EXTDATA2G)
  if (NOT TARGET YAFYAML::yafyaml)
     find_package(YAFYAML REQUIRED)
  endif ()
  set (EXTDATA2G_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData2G Target")
  message (STATUS "Building with ExtData2G")
else()
  set (EXTDATA2G_TARGET "" CACHE STRING "ExtData2G Target")
  message (STATUS "Building with ExtData1G")
endif()
```

This removes a bit of duplication (since `EXTDATA_TARGET` seems to always be defined...and do we even need a variable then?), and brings back the "If building like GCHP" bit about `if not target`.

Reply:
Yeah, you are right, we don't need the EXTDATA_TARGET anymore, I removed it